lib/std/math/log_int.zig

```diff
@@ -0,0 +1,81 @@
+const std = @import("../std.zig");
+const math = std.math;
+const testing = std.testing;
+const assert = std.debug.assert;
+const Log2Int = math.Log2Int;
+
+/// Returns the logarithm of `x` for the provided `base`, rounding down to the nearest integer.
+/// Asserts that `base > 1` and `x != 0`.
```

Comment:
Make it `x > 0` then, if log_int is intended to only support unsigned integers.

Reply:
Changed: 8a298473c